cli/dts/lib.esnext.array.d.ts

```diff
@@ -21,14 +21,60 @@ interface Array<T> {
    * @param index index to access.
    */
   at(index: number): T | undefined;
-}
+
+  /**
+   * Returns the value of the last element in the array where predicate is true, and undefined
+   * otherwise.
+   * @param predicate find calls predicate once for each element of the array, in ascending
+   * order, until it finds one where predicate returns true. If such an element is found, find
+   * immediately returns that element value. Otherwise, find returns undefined.
+   * @param thisArg If provided, it will be used as the this value for each invocation of
+   * predicate. If it is not provided, undefined is used instead.
+   */
+  findLast<S extends T>(predicate: (this: void, value: T, index: number, obj: T[]) => value is S, thisArg?: any): S | undefined;
+  findLast(predicate: (value: T, index: number, obj: T[]) => unknown, thisArg?: any): T | undefined;
```

Comment:
you need to repeat the comment block on the overload for hover to work property.

Reply:
Copied this from lib.es2015.core.d.ts - that has no two comments. Want me to change it anyway?

Reply:
naw... keeps it consistent then...